src/app/item-page/bitstreams/request-a-copy/bitstream-request-a-copy-page.component.html

```diff
@@ -64,22 +64,28 @@ <h1 class="mb-4">{{'bitstream-request-a-copy.header' | translate}}</h1>
                             for="message">{{'bitstream-request-a-copy.message.label' | translate}}</label>
                     <textarea rows="5"
                               [className]="'form-control'"
-                              id="message" formControlName="message"></textarea>
+                              id="message" formControlName="message" (input)="changeCatch()"></textarea>
                 </div>
             </div>
+            
         </div>
     </form>
-
+    <div class="row mb-4">
```

Comment:
This page is *missing the warning* to that you must first allow the Google Recaptcha cookie in your browser (in the "Cookie Settings" in the footer).  Take a look at how the register-email-form.component.html has a `<ds-alert>` : https://github.com/DSpace/dspace-angular/blob/main/src/app/register-email-form/register-email-form.component.html#L39-L42

This same sort of alert MUST exist on this page, as we cannot add the Captcha cookie unless users accept it in the Klaro cookie settings.